apps/app_rpt.c

```diff
@@ -2695,8 +2695,12 @@ static int rpt_setup_channels(struct rpt *myrpt, struct ast_format_cap *cap)
 		myrpt->txchannel = myrpt->rxchannel;
 		myrpt->dahditxchannel = IS_DAHDI_CHAN_NAME(myrpt->rxchanname) && !IS_PSEUDO_NAME(myrpt->rxchanname) ? myrpt->txchannel : NULL;
 	}
+	/* If the TX channel is not a pseudo channel, toggle the PTT line */
 	if (!IS_PSEUDO(myrpt->txchannel)) {
+		/* Wait for the driver to finish initialization */
```

Comment:
Why is 250 ms enough to finish initialization? Is there a more direct way to wait for initialization to finish? How do we know that has happened?